src/wp-admin/js/inline-edit-post.js

```diff
@@ -100,27 +148,36 @@ inlineEditPost = {
 			}
 		});
 
+		// If no boxes where checked, just hide the quick/bulk edit rows.
 		if ( c ) {
 			return this.revert();
 		}
 
+		// Add onclick events to the delete-icons for the post titles.
```

Comment:
This comment is somewhat unclear. What are the delete-icons and how do they relate to the post titles?